src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -471,20 +485,25 @@ private async Task ExecuteInitPs1Async(string installPath, PackageIdentity ident
                     AddPathToEnvironment(toolsPath);
 
                     var scriptPath = Path.Combine(toolsPath, PowerShellScripts.Init);
-                    if (File.Exists(scriptPath) &&
-                        _scriptExecutor.Value.TryMarkVisited(identity, PackageInitPS1State.FoundAndExecuted))
+                    if (File.Exists(scriptPath))
                     {
-                        // always execute init script on a background thread
-                        await TaskScheduler.Default;
+                        // Record if init.ps1 is loaded.
+                        NuGetPowerShellUsage.RaisInitPs1LoadEvent(_activeConsole is IWpfConsole);
```

Comment:
Add named argument to clarify parameter
```suggestion
                        NuGetPowerShellUsage.RaisInitPs1LoadEvent(isPMC: _activeConsole is IWpfConsole);
```

Reply:
Thank you for your review.
Implemented your suggestion.